octoprint_mrbeam/iobeam/dust_manager.py

```diff
@@ -70,6 +75,11 @@ def __init__(self, plugin):
         self._fan_timers = []
         self._last_command = dict(action=None, value=None)
         self._just_initialized = False
+        self._fan_not_spinning_ts = None
+        self._fan_data_missing_ts = None
+        self._fan_data_missing_reported = None
+        self._fan_data_to_old_reported = None
+        self._fan_data_spinning_reported = None
```

Comment:
just for naming consistency with the variables above
```suggestion
        self._fan_not_spinning_reported = None
```